### PR TITLE
add entry for 5.6.9 release

### PR DESCRIPTION
**Pull request by jburel**
Add entry for release

### CODE REVIEW

File: history.rst [hunk 1]
```diff
@@ -5,6 +5,39 @@
...
+
+Improvements include:
+
+- remove support for ADH and let Ice choose the default SSL protocol.
+- revert "Java client init: no "anon" among jdk.tls.disabledAlgorithms" 
+- an upgrade of Bio-Formats to version 7.0.0
+
+**Note:** This upgrade will invalidate the Bio-Formats Memoizer cache. Please
+see the upgrade guide for further information.
+We also recommend to re-index the database especially if the OMERO server has a large number of High Content Screening data.
```
Comment (sbesson) [inline history.rst]: Is this truly recommended on every upgrade?

**Review by sbesson — COMMENTED** [review]
Looking at 
https://github.com/ome/omero-model/blob/master/CHANGELOG.md#5612-september-2023
https://github.com/ome/omero-common/blob/master/CHANGELOG.md#563-september-2023
https://github.com/ome/omero-romio/blob/master/CHANGELOG.md#574-september-2023
https://github.com/ome/omero-renderer/blob/master/CHANGELOG.md#5514-september-2023
https://github.com/ome/omero-server/blob/master/CHANGELOG.md#569-september-2023
https://github.com/ome/omero-blitz/blob/master/CHANGELOG.md#571-september-2023
https://github.com/ome/omero-gateway-java/blob/master/CHANGELOG.md#582-september-2023

the Bio-Formats 7.0.0 upgrade and the Java client changes are the main Java changes which are neither dependency version bump nor mailmap updates and not worth mentioning in user-facing notes).

File: history.rst (hunk 1 above)
Comment (jburel) [reply history.rst]: No
This is a cut & paste I missed it

**Comment by jburel** [comment]
> Looking at https://github.com/ome/omero-model/blob/master/CHANGELOG.md#5612-september-2023 https://github.com/ome/omero-common/blob/master/CHANGELOG.md#563-september-2023 https://github.com/ome/omero-romio/blob/master/CHANGELOG.md#574-september-2023 https://github.com/ome/omero-renderer/blob/master/CHANGELOG.md#5514-september-2023 https://github.com/ome/omero-server/blob/master/CHANGELOG.md#569-september-2023 https://github.com/ome/omero-blitz/blob/master/CHANGELOG.md#571-september-2023 https://github.com/ome/omero-gateway-java/blob/master/CHANGELOG.md#582-september-2023
> 
> the Bio-Formats 7.0.0 upgrade and the Java client changes are the main Java changes which are neither dependency version bump nor mailmap updates and not worth mentioning in user-facing notes).

This is the approach we took after the split. 
We can change that if we think it is no longer needed
We only focus on key changes.

**Review by sbesson — APPROVED** [review]

**Pull request merged**